### PR TITLE
Added ChromeOS, Linux GS

### PR DESCRIPTION
**Pull request by atsansone**
Adds ChromeOS, Linux GS pages plus all 2nd level add-ons to getting started.

Fixes https://github.com/flutter/website/issues/10255
Fixes https://github.com/flutter/website/issues/10186
Fixes https://github.com/flutter/website/issues/10188
Fixes https://github.com/flutter/website/issues/10191

----

### ChromeOS install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/chromeos/
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/chromeos/android/
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/chromeos/web/

### Linux Install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/linux/
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/linux/desktop
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/linux/android
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/get-started/install/linux/web

### Add Linux desktop to Linux Android or Web install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/linux/install-linux/
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/linux/install-linux/install-linux-from-android
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/linux/install-linux/install-linux-from-web

### Add Android to Linux Desktop install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/android/install-android/install-android-from-linux
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/android/install-android/install-android-from-web-on-linux

### Add Android to ChromeOS Web install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/android/install-android/install-android-from-web-on-chromeos

### Add Web  to Linux Desktop install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/web/install-web/install-web-from-linux-desktop
- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/web/install-web/install-web-from-android-on-linux

### Add Web to ChromeOS Android install

- https://flutter-docs-prod--pr10302-fix-10255-q3e2r9ic.web.app/platform-integration/web/install-web/install-web-from-android-on-chromeos

----

## Refactors

- Flutter file replacement into command lines on all install pages
- Flutter doctor results for all combinations

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by anderdobo** [comment]
LGTM. Thanks!

**Comment by sfshaza2** [comment]
OK, this really should have had a writer review, but I don't have time right now.

File: src/platform-integration/android/install-android/install-android-from-web-on-linux.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Add Android devtools for Flutter from Web on Linux start
```
Comment (sfshaza2) [inline src/platform-integration/android/install-android/install-android-from-web-on-linux.md:2]: I am exceedingly confused by what this means. First of all, "devtools" is not a word. Flutter and Dart and Flutter (or Dart) DevTools. Chrome has Chrome DevTools. There, so far as I know, aren't any "web DevTools". I don't even understand this title.

File: src/platform-integration/linux/install-linux/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Add Linux devtools for Flutter
```
Comment (sfshaza2) [inline src/platform-integration/linux/install-linux/index.md:2]: What is Linux devtools? That is not a thing. There are Chrome DevTools and Dart/Flutter DevTools. I don't even understand what this means.

File: src/platform-integration/linux/install-linux/install-linux-from-android.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Add Linux devtools to Flutter from Android start
```
Comment (sfshaza2) [inline src/platform-integration/linux/install-linux/install-linux-from-android.md:2]: Ditto to above confusion.

File: src/platform-integration/linux/install-linux/install-linux-from-web.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: Add Linux devtools to Flutter from web start
```
Comment (sfshaza2) [inline src/platform-integration/linux/install-linux/install-linux-from-web.md:2]: Ditto.

File: src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Add Web devtools to Flutter from Android on ChromeOS start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-chromeos.md:2]: Ditto. No such thing as "Web devtools".

File: src/platform-integration/web/install-web/install-web-from-android-on-linux.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: Add Web devtools to Flutter from Android on Linux start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-android-on-linux.md:2]: Ditto.

File: src/platform-integration/web/install-web/install-web-from-linux-desktop.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+---
+title: Add Web devtools to Flutter from Linux start
```
Comment (sfshaza2) [inline src/platform-integration/web/install-web/install-web-from-linux-desktop.md:2]: Ditto

**Review by sfshaza2 — COMMENTED** [review]
Other than the confusion about "devtools", which really does confuse me, this _seems_ ok. It's very hard to evaluate changes that are mostly infrastructure and include files.

But the "xxx devtools" language should be cleaned up.

**Pull request merged**